pycbc/events/eventmgr.py

```diff
@@ -190,6 +191,33 @@ def __init__(self, opt, column, column_types, **kwds):
         self.template_events = numpy.array([], dtype=self.event_dtype)
         self.write_performance = False
 
+    def save_state(self, tnum_finished, filename):
+        """Save the current state of the background buffers"""
+        from six.moves import cPickle
+        from pycbc.io.hdf import dump_state
```

Comment:
Is there a need for a local import here? We don't normally do this unless it's an optional dependency, which these are not.

Reply:
It could be avoided for Pickle, but it's helpful to reduce cyclic imports for pycbc.io